src/Squirrel/UpdateManager.CheckForUpdates.cs

```diff
@@ -25,6 +25,7 @@ public CheckForUpdateImpl(string rootAppDirectory)
                 string localReleaseFile,
                 string updateUrlOrPath,
                 string token,
+                int maxDeltas,
```

Comment:
I know that Squirrel does not usually bother with any comments, but since this is our own param, I suggest documenting it also here.

Reply:
Ok, I've added constructor comment.